src/Driver/Configurations.jl

```diff
@@ -131,31 +110,15 @@ function GCM_Configuration(name::String,
                            init_GCM!;
                            array_type         = CLIMA.array_type(),
                            solver_type        = DefaultSolverType(),
-                           orientation        = SphericalOrientation(),
-                           T_min              = FT(200),
-                           T_surface          = FT(280),
-                           Γ                  = FT(grav) / FT(cp_d),
-                           rel_hum            = FT(0),
-                           ref_state          = HydrostaticState(LinearTemperatureProfile(T_min,
-                                                                                          T_surface,
-                                                                                          Γ),
-                                                                 rel_hum),
-                           C_smag             = FT(0.21),
-                           turbulence         = SmagorinskyLilly{FT}(C_smag),
-                           moisture           = EquilMoist(),
-                           precipitation      = NoPrecipitation(),
-                           radiation          = NoRadiation(),
-                           subsidence         = NoSubsidence{FT}(),
-                           sources            = (Gravity(), Coriolis()),
-                           bc                 = NoFluxBC(),
+                           model              = AtmosModel{FT}(;orientation=SphericalOrientation(),
+                                                                sources=(Gravity(), Coriolis()),
+                                                                init_state=init_GCM!),
```

Comment:
Then this would just use `model = AtmosModel{FT}(AtmosGCMConfiguration())`.

Reply:
Yes, but we'll still need the `init_state`, for now